addons/account/wizard/account_resequence.py

```diff
@@ -127,6 +127,9 @@ def resequence(self):
         # Can't change the name of a posted invoice, but we do not want to have the chatter
         # logging 3 separate changes with [state to draft], [change of name], [state to posted]
         self.with_context(tracking_disable=True).move_ids.state = 'draft'
+        if self.move_ids.journal_id and self.move_ids.journal_id.restrict_mode_hash_table:
+            if self.ordering == 'date':
+                raise UserError(_('You could not reorder sequence by date when the journal is locked with a hash.'))
```

Comment:
Shouldn't we check that the numbers (field `sequence_number`) and the prefix (field `sequence_prefix`) don't change somehow?
Adding a zero padding shouldn't change these fields.
Maybe adding it in the hash would do the trick...

Reply:
The error message should be, "you can not reorder sequence by date when the journal is locked with a hash". Also when adding a translatable string in a stable version, you should add it in the .pot file so that we can ask for it to be translated :)